index.html

```diff
@@ -0,0 +1,386 @@
+                  src="images/Counter.png" alt="counter"></span><span class="inrow1"> 2015</span>
+            </p><br>
+            <p class="innerpara1">Experimental content creation feature that allows users to add to an existing story
+              over the course of a day without spamming their friends.
+            </p><br>
+            <ul>
+              <li class="back4">html</li>
+              <li class="back4">Ruby on rails</li>
+              <li class="back4">css</li>
+              <li class="back4">javascript</li>
+            </ul><br>
+            <button type="button" class="button button4">See Project</button>
+          </div>
+          <div class="workimage1">
+            <img src="images/Snapshoot-Portfolio2.png" alt="2nd work immage">
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="workimage">
+            <img src="images/Snapshoot1.png" alt="2nd work immage">
+            <h2 class="tonic">Multi-Post</h2>
+            <h2 class="tonic"> Stories</h2> <br>
+            <p>
+              <span class="inrow"><b>CANOPY </b></span><span class="inrow"><img src="images/Counter.png"
+                  alt="counter"></span><span class="inrow"> Back End Dev </span><span class="inrow"><img
+                  src="images/Counter.png" alt="counter"></span><span class="inrow"> 2015</span>
+            </p><br>
+            <p class="innerpara">A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p><br>
+            <ul>
+              <li class="back">html</li>
+              <li class="back">css</li>
+              <li class="back">javascript</li>
+            </ul><br>
+            <button type="button" class="button button4">See Project</button>
+          </div>
+        </div>
+
+        <div class="card1">
+          <div class="workimage1">
+            <img src="images/Snapshoot-Portfolio3.png" alt="4th work immage">
+          </div>
+          <div class="sidepart1">
+            <h2 class="tonic1">FACEBOOK 360</h2>
+            <br>
+            <p>
+              <span class="inrow1"><b>FACEBOOK </b></span><span class="inrow1"><img src="images/Counter.png"
+                  alt="counter1"></span><span class="inrow1"> Full Stack Dev </span><span class="inrow1"><img
+                  src="images/Counter.png" alt="counter"></span><span class="inrow1"> 2015</span>
+            </p><br>
+            <p class="innerpara1">Experimental content creation feature that allows users to add to an existing story
+              over the course of a day without spamming their friends.
+            </p><br>
+            <ul>
+              <li class="back4">html</li>
+              <li class="back4">Ruby on rails</li>
+              <li class="back4">css</li>
+              <li class="back4">javascript</li>
+            </ul><br>
+            <button type="button" class="button button4">See Project</button>
+          </div>
+        </div>
```

Comment:
- [ ] Consider to match your design like Figma and make image align in right

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/173160844-3fc8f43e-edfa-4ad4-a0a7-5b1b605cd4a8.png) | ![image](https://user-images.githubusercontent.com/53402600/173160894-c418637f-7b8c-4298-8eea-62ed783705d1.png) |